includes/User.php

```diff
@@ -4894,9 +4894,8 @@ private function loadAttributes() {
 			foreach ( $attributes as $attributeName => $attributeValue ) {
 				$this->compareAttributeValueFromService( $attributeName, $attributeValue );
 
-				// Once shadow mode verifies these are correct, we can uncomment this
-				// $this->mOptionOverrides[$attributeName] = $attributeValue;
-				// $this->mOptions[$attributeName] = $attributeValue;
+				 $this->mOptionOverrides[$attributeName] = $attributeValue;
+				 $this->mOptions[$attributeName] = $attributeValue;
```

Comment:
@jsutterfield: Should this be controlled via a wg variable as discussed in the previous ticket [SOC-1216](https://wikia-inc.atlassian.net/browse/SOC-1216?focusedCommentId=193377&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-193377)?